src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/client/CommunityApiClient.kt

```diff
@@ -54,6 +54,8 @@ class CommunityApiClient(@Qualifier("communityWebClientAppScope") private val we
 
   fun getRequirements(crn: String, convictionId: Long): List<Requirement> {
     return getRequirementsCall(crn, convictionId)
+      .filterNot { it.requirementTypeMainCategory == null && it.restrictive == null }
+      .map { Requirement(it.restrictive!!, it.requirementTypeMainCategory!!.code) }
```

Comment:
It looks like our service layer is emerging in the client, and we're creating a domain separate from the DTOs